siliconcompiler/schema/schema_cfg.py

```diff
@@ -1038,7 +1043,8 @@ def schema_flowgraph(cfg, flow='default', step='default', index='default'):
             * "success": task ran successfully
             * "error": task failed with an error
 
-            An empty value indicates the task has not yet been completed.""")
+            An empty value indicates the task has not yet been completed.""",
```

Comment:
Can we put this right below type for readability? Feels like the enum list belongs with the type since it's static.

Reply:
I'll move them up in the _cfg.py